src/integration/pages/onDemandTVPage/pashto/__snapshots__/amp.test.js.snap

```diff
@@ -122,10 +122,46 @@ exports[`AMP On Demand T V Page Footer I can click on the BBC branding and it wo
 
 exports[`AMP On Demand T V Page Footer I can see the footer copyright and external linking text 1`] = `"© 2020 بي بي سي. بي بي‌ سي‌ د نورو ویبپاڼو د محتوا مسوله نه ده. د نورو ویبپاڼو لینکولو په اړه زموږ تګلاره."`;
 
+exports[`AMP On Demand T V Page Footer Link should match text and url 1`] = `"BBC News, پښتو - /pashto"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 2`] = `"ولې تاسې پر بي بي سي خبرونو باور کولی شئ - https://www.bbc.com/pashto/institutional-49283007"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 3`] = `"د کارولو شرايط - https://www.bbc.com/pashto/institutional-37620701"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 4`] = `"د بي بي سي‌ په اړه - https://www.bbc.com/pashto/institutional-37622761"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 5`] = `"د محرميت تگلاره - https://www.bbc.com/pashto/institutional-37622762"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 6`] = `"Cookies - https://www.bbc.com/usingthebbc/cookies/"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 7`] = `"زموږ سره اړیکي - https://www.bbc.co.uk/pashto/institutional-37622768"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 8`] = `"AdChoices / Do Not Sell My Info - https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/"`;
+
+exports[`AMP On Demand T V Page Footer Link should match text and url 9`] = `"د نورو ویبپاڼو لینکولو په اړه زموږ تګلاره. - https://www.bbc.co.uk/help/web/links/"`;
+
```

Comment:
@karinathomasbbc  can you confirm if this snapshot updates should have been added in this [PR](https://github.com/bbc/simorgh/pull/6282)

Reply:
Yes, that looks right

Reply:
Actually it should have been in https://github.com/bbc/simorgh/pull/6733

Reply:
Thanks, I'll go ahead and merge this in then.